install/kubernetes/helm/istio/requirements.yaml

```diff
@@ -3,43 +3,68 @@ dependencies:
     version: 0.8.0
     # repository: file://../sidecar-injector
     condition: sidecar-injector.enabled
+    tags:
+      - integrated_citadel
```

Comment:
How about using `standalone_citadel` as a tag? Then you don't need to explicitly set `integrated_citadel` tags here.

Reply:
I'm not convinced this PR works for the non-standalone case - need a bit of help from @jascott1.

Unfortunately standalone_citadel, while making more logical sense, is not viable with the way Helm works.  The way tags and conditions are processed is covered here: https://github.com/kubernetes/helm/blob/master/docs/charts.md#tags-and-condition-fields-in-requirementsyaml

In short here is my parsing of how conditions/tags work (or don't work) together:
If subchart.condition == undefined
  if tag == true
     include subchart

There is no way to have the opposite logic:

if subchart.condition == undefined
  if tag = true (on the security)
     include all charts that are not security

@jascott1 can you weigh in here is this PR correct, or is there a way to specify "standalone_citadel".

I get the impression condition/tags are half-baked after reading the docs, and need some more time in the oven ;-)